doc/security/security-overview.rst

```diff
@@ -31,9 +31,9 @@ in detail. As depicted in Figure 1, these main steps are:
    to enforce them. A security architecture of the system and
    relevant sub-modules is created, threats are identified, and
    countermeasures designed. Their correct implementation and the
-   validity of the threat models are checked by code reviews.
+   validity of the threat models is checked by code reviews.
```

Comment:
not about this one, it was fine before?

Reply:
Oops, sorry. Fixed now!

Also unified the spelling of _Mbed TLS_.